### PR TITLE
📝 Update about page

### PR DESCRIPTION
**Pull request by dhelle2**
paragraph 2- "Bridge" -> means of communication

ACM -> ACM@UIC in paragraph 3
mentions documentation pages and blog and also GBM (also paragraph 3)

### CODE REVIEW

File: src/pages/about.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
 activities with other tech related clubs and an office to hang out in and talk or study adjacent to the cs lounge
 on the second floor of SELE on campus. Our discord and slack servers also provide virtual communities online. Additionally we have a number of special interest groups (SIGs) which meet regularly to discuss and work on specific topics
 and projects.
 
-ACM also provides computing resources such as servers, software and a working network in our office, accounts for
+ACM@UIC provides computing resources such as servers, software and a working network in our office, accounts for
 members along with the opportunity to obtain practice maintaining and improving these resources through
-activities and SIGs. See the following pages for more information and documentation.
+activities and SIGs. See the Documentation pages and the blog or watch for and attend the General Body Meeting regularly scheduled
+near the beginnig of each semester.
```
Comment (clee231) [inline src/pages/about.md:13]:
```suggestion
near the beginning of each semester.
```
You have a typo here.

**Review by clee231 — CHANGES_REQUESTED** [review]
Some sentences are broken up in odd ways. Can we also update this to be consistent? Either have each paragraph as a single line or add natural line breaks around the 80 or 100 column mark. ([To Wrap or Not to Wrap: One of Life's Greatest Questions](https://about.gitlab.com/blog/2016/10/11/wrapping-text/))

File: src/pages/about.md (hunk 1 above)
Comment (dhelle2) [reply src/pages/about.md]: I think I fixed it all - let me know if not (or if you don't see the changes)

Comment (clee231) [reply src/pages/about.md]: This looks good @dhelle2! Approved!

**Review by clee231 — APPROVED** [review]

**Review by bmiddha — APPROVED** [review]

**Pull request merged**